### ISSUE TITLE
Ambassador Task: Create a video about Horizen’s ecosystem

### ISSUE THREAD

**Issue by PowerVANO** [HorizenOfficial/HDE-Proposals#9]
**Goals:** 
- Create a short video highlighting your favorite aspects of the Horizen ecosystem (such as Sphere by Horizen, HDE, etc)
- Share the video on your preferred platform 

**Scope of work:** 

This video should be 5 minutes or less and must cover at least one of your favorite aspects of the Horizen ecosystem. 

**Schedule Milestones:** 

This task may only be completed by an **official Horizen Ambassador**. This task should be completed within 4 weeks. If a user does not complete this task within the allotted time, it will be considered open for another user to complete. For the task to be considered complete, it will need to be reviewed and approved by the Horizen team. The team may request edits be made to the video before approval. 

Ambassadors that accept this task must provide weekly updates during the period they are working on the task. 

**Budget: $100 (Paid in ZEN)**

**Comment by myphuong776** [comment]
ok

**Comment by teysonGold** [comment]
Puedo asumir esa tarea, pero en español o que idioma lo necesitan?

**Comment by teysonGold** [comment]
hola, cuanto tiempo dura la aprobación para ser embajador oficial?

**Comment by PowerVANO** [comment]
@horusfeathers Can you please help here?

**Comment by Horusfeathers** [comment]
Todas las solicitudes han sido procesadas. Por favor revise su correo electrónico.

**Comment by HaulMAD** [comment]
@PowerVANO would appreciate the opportunity to take the task

**Comment by Horusfeathers** [comment]
> @PowerVANO would appreciate the opportunity to take the task

Please apply to the Horizen Ambassador Program to potentially become eligible for this task. 

https://www.horizen.io/ambassadors/

**Comment by pvmihalache** [comment]
My ZEN ELI5 will be ready soon

**Comment by pvmihalache** [comment]
> My ZEN ELI5 will be ready soon

https://odysee.com/@MindPuzzles:9/ZEN:66

https://youtu.be/3ucq6kfQMAI

**Comment by Sam-Steel55** [comment]
Hello, I would like to take on this task.

**Comment by Sam-Steel55** [comment]
I would like to take on this task.

**Comment by Horusfeathers** [comment]
> 

Sam Steel is approved to complete this task @PowerVANO

**Comment by PowerVANO** [comment]
@Sam-Steel55 Please keep us posted about the task progress according to the task description.

**Comment by toprakdere** [comment]
i have done video for turkish community https://www.youtube.com/watch?v=dpvutu4V07o

**Comment by Sam-Steel55** [comment]
> 

I have created a draft for this task and submitted it to @Horusfeathers. Still awaiting review.

**Comment by Sam-Steel55** [comment]
Posted my video! ⤵️⤵️

https://twitter.com/_Sam_Steel/status/1524571970872000514?t=12q7AMW4wsYXLj399Pnc4A&s=19

**Comment by Sam-Steel55** [comment]
> 

Posted my video! ⤵️⤵️

https://twitter.com/_Sam_Steel/status/1524571970872000514?t=12q7AMW4wsYXLj399Pnc4A&s=19

**Comment by PowerVANO** [comment]
@Sam-Steel55 

@Horusfeathers approved your work and you have been paid.
